values-schema.yaml

```diff
@@ -408,7 +417,7 @@ definitions:
       accountJson:
         type: string
         x-secret: ''
-      projectId:
+      project:
```

Comment:
Looks like property rename. If so please add it to the `values-changes.yaml` file?

Reply:
of course